crates/subspace-farmer/src/single_plot_farm.rs

```diff
@@ -739,3 +780,23 @@ impl<RC: RpcClient> OnSync for VerifyingPlotter<RC> {
             .map_err(Into::into)
     }
 }
+
+/// Pieces verification errors.
```

Comment:
This is minor, but I try to structure the code such that it starts with table of contents in a away with the main content following. So I'd put `PiecesVerificationError` right above `VerifyingPlotter` struct definition (definition, not implementation) because you'll need to know what this enum looks like once you hit `Result<(), PiecesVerificationError>` a few lines below. Moving it to the very bottom of the file almost 100 lines lower makes you jump back and forth over the file when reading. Having things closer helps.